R/dist.R

```diff
@@ -316,17 +316,31 @@ lognorm_dist_def <- function(mean, mean_sd,
                              max_value, samples,
                              to_log = FALSE) {
   
+  transform_mean <- function(mu, sig){
+    mean_location <- log(mu^2 / sqrt(sig^2 + mu^2))
+    mean_location
+    }
+  
+  transform_sd <- function(mu, sig){
+    mean_shape <- sqrt(log(1 + (sig^2 / mu^2)))
+    mean_shape
+    }
+  
+  sampled_means <- truncnorm::rtruncnorm(samples, a = 0, mean = mean, sd = mean_sd)
```

Comment:
can we just clean this up so samles goes -> means without the interim step? Saves the reallocation on the lines below.